File: api/game/presentation/proto/health/health_check_response.gen.go

// Package health ヘルスチェックレスポンス
package health

import (
	
"github.com/game-core/gc-server/api/game/presentation/proto/health/adminHealth"
"github.com/game-core/gc-server/api/game/presentation/proto/health/adminHealth"
"github.com/game-core/gc-server/api/game/presentation/proto/health/commonHealth"
"github.com/game-core/gc-server/api/game/presentation/proto/health/commonHealth"
"github.com/game-core/gc-server/api/game/presentation/proto/health/masterHealth"
"github.com/game-core/gc-server/api/game/presentation/proto/health/masterHealth"
)

type HealthCheckResponses []*HealthCheckResponse

func NewHealthCheckResponse() *HealthCheckResponse {
			return &HealthCheckResponse{}
		}

		func NewHealthCheckResponses() HealthCheckResponses {
			return HealthCheckResponses{}
		}

		func SetHealthCheckResponse(adminHealth *adminHealth.AdminHealth,commonHealth *commonHealth.CommonHealth,masterHealth *masterHealth.MasterHealth) *HealthCheckResponse {
			return &HealthCheckResponse{
				AdminHealth: adminHealth,
CommonHealth: commonHealth,
MasterHealth: masterHealth,
			}
		}
		

File: view/admin/pkg/domain/model/health/HealthCheckResponse.gen.ts
import type { AdminHealth } from "~/pkg/domain/model/health/adminHealth/AdminHealth.gen";
import type { CommonHealth } from "~/pkg/domain/model/health/commonHealth/CommonHealth.gen";
import type { MasterHealth } from "~/pkg/domain/model/health/masterHealth/MasterHealth.gen";

export type HealthCheckResponse = {
  adminHealth: AdminHealth;
  commonHealth: CommonHealth;
  masterHealth: MasterHealth;
};
